nnoir/nnoir/nntc.py

```diff
@@ -1,66 +1,68 @@
 import io
+from typing import Any, Dict, List
 
 import msgpack
 import numpy as np
+from numpy.typing import NDArray
 
 
-class NNTC:
-    def __init__(self, model_name, inputs, outputs):
-        self.version = 0
-        self.model = model_name
-        self.test_case = TestCase(inputs, outputs)
-
-    def to_dict(self):
-        return {
-            b"version": self.version,
-            b"model": self.model,
-            b"test_case": self.test_case.to_dict(),
-        }
+class TestValue:
+    def __init__(self, name: bytes, ndarray: NDArray[Any]):
+        self.name = name
+        self.ndarray = ndarray
```

Comment:
Rearrange the order of class definitions (`NNTC`, `TestValue`, and `TestCase`) to avoid mypy name resolution errors.